gluonnlp/vocab/subwords.py

```diff
@@ -162,6 +168,62 @@ def subwords_to_indices(self, subwords):
         return subwords
 
 
+@numba_njit
+def _byte_to_int(b):
+    return b
+
+
+if sys.version_info[0] == 2:
+    # Python 2 requires an extra `ord()` when operating on memoryview outside of numba
+
+    try:
+        import numba
+        if numba.config.DISABLE_JIT:
+            # JIT disabled is equivalent to numba not being present
+            raise ImportError
+    except ImportError:
+
+        # pylint: disable=function-redefined
+        @numba_njit
+        def _byte_to_int(b):
+            return ord(b)
```

Comment:
would it make sense for this to be in base too?

Reply:
I'm not sure if it would be needed anywhere else